plugins/woocommerce-docs/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "woocommerce-docs",
+	"version": "1.0.0",
+	"description": "WooCommerce Docs Plugin",
+	"main": "index.js",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"build": "wp-scripts build"
+	},
```

Comment:
either add a `postinstall: composer install` step or add the step to your directions.

Reply:
Good point thank you. 🤔 probably to keep consistency with the rest of the monorepo a postinstall step makes a lot of sense here.

Reply:
Done here https://github.com/woocommerce/woocommerce/pull/38406/commits/820478b11281ef931512b757e58872a036dbd7b5